instaclustr/resource_azure_peering.go

```diff
@@ -0,0 +1,178 @@
+package instaclustr
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAzureVpcPeering() *schema.Resource {
+	return &schema.Resource{
+		Create: AzureresourceVpcPeeringCreate,
+		Read:   AzureresourceVpcPeeringRead,
+		Update: resourceVpcPeeringUpdate,
+		Delete: resourceVpcPeeringDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: resourceVpcPeeringStateImport,
+		},
+
+		Schema: map[string]*schema.Schema{
+
+			"cluster_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"cdc_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"peer_resource_group": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"peer_suscription_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"peer_vpc_net": {
```

Comment:
I know on the API side of things, some of this is abbreviated - should double check though and make sure that we're consistent

Reply:
I get the argument for consistency, though I have the preference of having a better name for this on the terraform schema because the API property name isn't great. 

Thinking about this a bit more - I don't think we can assume that terraform users know the API schema by heart to make the comparison whenever they read the terraform code. Also if there's a concern about the new name being confusing because it doesn't match the API, we could cover that in the documentation.